src/util/options.schema.json

```
@@ -1364,6 +1364,12 @@
             "Sets the unrolling factor for the loopUnrollingVisitor.",
           "type": "integer",
           "default": 0
+        },
+        "hide-std-globals": {
+          "title": "exp.hide-std-globals",
+          "description": "Hide standard extern globals from cluttering local states, e.g. stdout.",
```

```suggestion
          "description": "Hide standard extern globals (e.g. `stdout`) from cluttering local states.",
```
Would changing the description like make it a bit easier to parse?